### PR TITLE
YDA-5808 - Modified to display Base DOI in System Metadata and all versions of selected datapackage in VAULT space.

### PR DESCRIPTION
**Pull request by kaur16**
YDA-5808 - Modified to display Base DOI in System Metadata and all versions of selected datapackage in VAULT space.

### CODE REVIEW

File: vault.py [hunk 1]
```diff
@@ -485,14 +485,19 @@ def api_vault_system_metadata(ctx, coll):
         landinpage_url = row[0]
         system_metadata["Landingpage"] = "<a href=\"{}\">{}</a>".format(landinpage_url, landinpage_url)
 
+    # Check for base DOI
```
Comment (lwesterhof) [inline vault.py]: Lets discuss with the team, but maybe we can drop the DOIs from the system metadata if we always display the versions? Also when only a single version DOI exists?

File: vault.py [hunk 2]
```diff
@@ -1467,17 +1512,18 @@ def get_approver(ctx, path):
         return None
 
 
-def get_doi(ctx, path):
+def get_doi(ctx, path, doi):
```
Comment (lwesterhof) [inline vault.py]: Safer to add a default to doi: `doi="version"` and add a check to set it to version if it is not `base`?

```
def get_doi(ctx, path, doi="version"):
    if doi != "base":
        doi="version"
```

File: vault.py [hunk 3]
```diff
@@ -580,6 +585,43 @@ def api_vault_collection_details(ctx, path):
 
     basename = pathutil.basename(path)
 
+    # If specific data package is selected, retreive all published versions
```
Comment (lwesterhof) [inline vault.py]: Move this inside the "Check if collection is vault package" if else clause for a safer way to check this.

**Review by lwesterhof — CHANGES_REQUESTED** [review]

File: vault.py (hunk 1 above)
Comment (kaur16) [reply vault.py]: Removed DOIs from system metadata section.

**Review by lwesterhof — APPROVED** [review]
LGTM!

**Pull request merged**